core/src/main/java/com/sequenceiq/cloudbreak/service/template/ClusterTemplateService.java

```diff
@@ -154,7 +180,7 @@ private void validateBeforeCreate(ClusterTemplate resource) {
 
         if (clusterTemplateRepository.findByNameAndWorkspace(resource.getName(), resource.getWorkspace()).isPresent()) {
             throw new BadRequestException(
-                    String.format("clustertemplate already exists with name '%s' in workspace %s", resource.getName(), resource.getWorkspace().getName()));
+                    format("clustertemplate already exists with name '%s' in workspace %s", resource.getName(), resource.getWorkspace().getName()));
```

Comment:
Not my typo, but going to fix it. Static import because we're using String.format in many places in the class, and it's doesn't make much difference but puts less reading effort to the dev who reads the code